pkgs/by-name/pi/pixivfe/package.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitea,
+  makeBinaryWrapper,
+}:
+buildGoModule rec {
+  pname = "pixivfe";
+  version = "2.5.1";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "VnPower";
+    repo = "PixivFE";
+    rev = "v${version}";
+    hash = "sha256-G2pSPpemMFAbQ9QkI4XAHobv+Em9ZoDUJiO/cwEy4Tc=";
+  };
+
+  vendorHash = "sha256-QapDR964Tn+RxXdkGqCQXacdmlSapF841Y84n4d/6VI=";
+
+  nativeBuildInputs = [ makeBinaryWrapper ];
+
+  # PixivFE require files from source code
+  postInstall = ''
+    wrapProgram $out/bin/pixivfe \
+      --chdir ${src}
```

Comment:
How about moving the views dir into $out and patching the main.go?

Reply:
I don't know. I just feel it's weird to rely on the src package.

Reply:
This adds a dependency on the source code. We normally do not want that. You likely want to land in $out and copy some missing files to $out/share/pixivfe or so